hack/ginkgo-e2e.sh

```diff
@@ -130,13 +128,12 @@ else
     "--auth_config=${AUTH_CONFIG:-}"
     "--cert_dir=${CERT_DIR:-}"
   )
-  host="https://${KUBE_MASTER_IP-}"
 fi
 
 # Use the kubectl binary from the same directory as the e2e binary.
 export PATH=$(dirname "${e2e}"):"${PATH}"
 "${e2e}" "${auth_config[@]:+${auth_config[@]}}" \
-  --host="$host" \
+  --host="https://${KUBE_MASTER_IP-}" \
```

Comment:
Yes, and checking whether `auth_config` contains `--auth_config` or `--kubeconfig` in order to add `--host` only in the former case would make things uselessly more complex.

I just added a comment to stress the fact that `--host` is not always used. Could you check if it’s OK for you?